packages/web-components/src/components/cta-section/__stories__/README.stories.mdx

```diff
@@ -17,32 +16,158 @@ Here's a quick example to get you started.
-import '@carbon/ibmdotcom-web-components/es/components/button-group/button-group.js';
-import '@carbon/ibmdotcom-web-components/es/components/button-group/button-group-item.js';
-
+import '@carbon/ibmdotcom-web-components/es/components/content-block/content-block-copy.js';
+import '@carbon/ibmdotcom-web-components/es/components/cta/link-list-item-cta.js';
+import '@carbon/ibmdotcom-web-components/es/components/video-player/video-player-container.js';
 ```
 
 ### HTML
 
 ```html
 <dds-cta-section>
-  <dds-content-block-heading>Curabitur malesuada varius mi eu posuere</dds-content-block-heading>
-  <dds-cta-section-copy>Lorem ipsum *dolor* sit amet</dds-cta-section-copy>
-  <dds-button-group slot='buttons'>
-    <dds-button-group-item href="https://example.com">
-      Primary Button
-    </dds-button-group-item>
-    <dds-button-group-item href="https://example.com">
-      Primary button
-    </dds-button-group-item>
-  </dds-button-group>
+  <dds-content-section-heading>Related products and services</dds-content-section-heading>
+
+  <dds-cta-block>
+    <dds-content-block-heading>DDS Content Block Heading</dds-content-block-heading>
+    <dds-content-block-copy>DDS Content Block Copy</dds-content-block-copy>
+    <dds-text-cta slot="action" cta-type="local" icon-placement="right" href="example.com">Browse tutorials</dds-text-cta>
+
+    <dds-cta-block-item-row no-border>
+      <dds-cta-block-item>
+        <span slot="statistics">10%</span>
+        <dds-content-item-heading>Get connected</dds-content-item-heading>
+        <dds-content-item-copy
+          >IBM DevOps partners have a wide range of expertise. Find one to build that right solution for
+          you.</dds-content-item-copy
+        >
+        <dds-text-cta slot="footer" cta-type="local" icon-placement="right" href="example.com">Find a partner</dds-text-cta>
+      </dds-cta-block-item>
+
+      <dds-cta-block-item>
+        <span slot="statistics">11%</span>
+        <dds-content-item-heading>Learn how</dds-content-item-heading>
+        <dds-content-item-copy>Dig into more self-directed larning about DevOps methodologies.</dds-content-item-copy>
+        <dds-text-cta slot="footer" cta-type="local" icon-placement="right" href="example.com">Browse tutorials</dds-text-cta>
+      </dds-cta-block-item>
+
+      <dds-cta-block-item>
+        <span slot="statistics">12%</span>
+        <dds-content-item-heading>Learn how</dds-content-item-heading>
+        <dds-content-item-copy>Dig into more self-directed larning about DevOps methodologies.</dds-content-item-copy>
+        <dds-text-cta slot="footer" cta-type="local" icon-placement="right" href="example.com">Browse tutorials</dds-text-cta>
+        <dds-text-cta slot="footer" cta-type="local" icon-placement="right" href="example.com">Browse tutorials</dds-text-cta>
+      </dds-cta-block-item>
+    </dds-cta-block-item-row>
+    <dds-cta-block-item-row>
+      <dds-cta-block-item>
+        <svg
```

Comment:
@RaphaelAmadeu it looks like the API for CTA Section has changed, can you add in as a breaking change under the pinned issue? We'll need to probably call this out in our announcement too...

Reply:
I'm wondering even if a note should be added to the README that the API for CTA Section has changed as of v1.18.0, thoughts @RaphaelAmadeu ?

Reply:
Perhaps add a section like this in the README:

```
## Breaking Changes
  
As of `v1.18.0`, the API for `<dds-cta-section>` has changed. Be sure to update to the latest version of CTA Section if upgrading from a previous version.
```